code/modules/mob/living/carbon/human/human.dm

```diff
@@ -49,6 +49,15 @@
 	return ..()
 
 /mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+
+	if(HAS_TRAIT(src, TRAIT_BAD_KNEES) && body_position == STANDING_UP) // uh oh
+		visible_message(span_danger("[src] makes a hard landing on [T] landing directly on their stiff legs! Ouch!"), \
+					span_userdanger("You land directly on your poor legs, sending a shockwave up your knees!"))
+		var/obj/item/bodypart/wrecked_leg_L = src.get_bodypart(BODY_ZONE_L_LEG)
+		var/obj/item/bodypart/wrecked_leg_R = src.get_bodypart(BODY_ZONE_R_LEG)
+		wrecked_leg_L.receive_damage(brute = 15, wound_bonus = 35)
+		wrecked_leg_R.receive_damage(brute = 15, wound_bonus = 35)
```

Comment:
pretty sure you can just do `apply_damage()` and pass `BODY_ZONE_R_LEG` as the defzone arg

Reply:
huh, no kidding.

Reply:
Oddly, this worked in testing. I'll still fix the code.